src/site/topics/inheritance/inheritance.rst

```diff
@@ -40,113 +44,137 @@ Terminology
 ===================
 
-.. image:: collections_inheritance.png
-   :width: 750 px
-   :align: center
-   :target: https://en.wikipedia.org/wiki/Java_collections_framework
+.. figure:: collections_inheritance.png
+    :width: 750 px
+    :align: center
+    :target: https://en.wikipedia.org/wiki/Java_collections_framework
+
+    Visualization of part of Java's collections' class hierarchy.
+
 
-* This is a visualization of part of Java's collections' class hierarchy
 
 Abstract Class
 --------------
 
-* You will notice that there is an *Abstract Class* designation
+* Notice the *Abstract Class* designation
 * An abstract class is one that
+
     * Is declared with the ``abstract`` keyword
     * May or may not include abstract methods
-        * Methods declared, but with no implementation, like interfaces
-    * May **not** be instantiated
+
+        * Method signatures, like interfaces
+
+
+    * Cannot be instantiated
     * May be subclassed/inherited from
+    * May include concrete methods that will be the same among multiple subclasses
 
-* An abstract class may include some concrete implementation of methods that will be the same among multiple subclasses
 
 * `Take Java's AbstractQueue class <https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/AbstractQueue.html>`_ and its subclass, `PriorityQueue <https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/PriorityQueue.html>`_
 
-* The ``AbstractQueue`` has a few important concrete methods implemented within the abstract class (it also inherits a bunch from superclasses too)
-    * ``add``
+* The ``AbstractQueue`` has several important concrete methods implemented within the abstract class
+
+    * It also inherits a bunch from superclasses too)
+
+
+* ``add``
+
     * ``addAll``
     * ``clear``
     * ``element``
     * ``remove``
 
+
 * The ``PriorityQueue`` class, which ``extends AbstractQueue``, makes direct use of a few of these methods
+
     * ``addAll``
     * ``element``
     * ``remove``
 
-* But the ``PriorityQueue`` also overrides a few of the methods from ``AbstractQueue`` since it needs a specific implementation
+
+* But the ``PriorityQueue`` also overrides a few of the methods from ``AbstractQueue`` since it requires a specific implementation
+
     * ``add``
     * ``clear``
 
+
 * If you open the ``PriorityQueue.java`` file
-    * You will not find ``addAll``, ``element``, or ``removed`` within it since it is inherited
-    * You will find the overridden methods (``add`` and ``clear``)
+
+    * One cannot find ``addAll``, ``element``, or ``removed`` within it since they are inherited
+    * One can find the overridden methods ``add`` and ``clear``
+
 
 
 Polymorphism
 ============
 
-* Polymorphism is the idea that behavior can change depending on the type of the object
-* We've already made use of this idea without having gone into any details
+* Polymorphism is the idea that behavior can change depending on the type of the *object*
```

Comment:
Inheritance is one form of polymorphism. The overall definition is accurate enough here, but another bullet point simply saying mentioning that polymorphism is broader than inheritance would be nice. I find most people fully equate the two concepts. Java specifically does support other polymorphisms even without type classes, like function overloading.